getimage.py

```diff
@@ -0,0 +1,48 @@
+import httplib, urllib, base64, random, string, json, pprint, cStringIO
+from PIL import Image
```

Comment:
Extraction of internal modules from a package should be ordered after **all** imports of full packages.